pom.xml

```diff
@@ -1120,12 +1124,20 @@
 							</defineStatements>
 							<preinstallScriptlet>
 								<script>if [ $1 = 1 ]; then
-                  USER_ID=`id -u %{package.user} 2&gt;/dev/null`
-                    if [ -z "$USER_ID" ]; then
+                                                                USER_ID=${dollar.sign}(id -u ${package.user} 2&gt;/dev/null)
+                                                                if [ -z "$USER_ID" ]; then
+                                                                REDHAT_RELEASE=${dollar.sign}(lsb_release -sr|cut -d"." -f1)
+                                                                if [ ${dollar.sign}{REDHAT_RELEASE} -gt 5 ]; then
 									    /usr/sbin/useradd -c "${project.name}" -s /bin/sh -r -d \
-									    ${package.install.dir} -U ${package.user}
-                    fi
-									fi</script>
+									    ${package.install.dir} -u ${package.userId} -U ${package.user}
```

Comment:
Seems we have inconsistent indentation. Could you make sure indentation is tab based and coherent?